plat/xen/arm/link32.lds.S

```diff
@@ -87,26 +87,7 @@ SECTIONS
 
 	TLS_SECTIONS
 
-	/* Data */
-	_data = .;
-	.data : {
-		*(.data)
-		*(.data.*)
-	}
-	_edata = .; /* End of data included in image */
-
-	/*
-	 * NOTE: linker will insert any extra sections here,
-	 * just before .bss
-	 */
-
-	.bss : {
-		__bss_start = .;
-		*(.bss)
-		*(.bss.*)
-	}
-	. = ALIGN(__PAGE_SIZE);
-	_end = . ;
```

Comment:
Does this line need to be deleted?
From what I saw from the link64.lds.S files, this line remained.

Reply:
Uuups, good that you saw this,  it should actually stay.